### PR TITLE
[Windows] Fixes version of the Python needed for LLDB to working properly

### PR DESCRIPTION
**Pull request by HeMet**
### Motivation:

It seems, Installation Guide for Windows has incorrect version of Python specified.
See https://forums.swift.org/t/unable-to-debug-on-windows/67609

### Modifications:

I've changed version of Python required from `3.10` to `3.9`

### Result:

The issue with debugger is solved.

### CODE REVIEW

File: install/windows/_winget.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
    ~~~ batch
    winget install Git.Git
-   winget install Python.Python.3.10
+   winget install Python.Python.3.9
```
Comment (compnerd) [inline install/windows/_winget.md:9]: I think that we should externalise this to a table.  The different releases of Swift require different versions.  If I'm not mistaken, 5.9.0 requires 3.11 🤦

Comment (compnerd) [reply install/windows/_winget.md:9]: Okay, verified that the 5.9 release binaries do need python39.dll.

**Comment by compnerd** [comment]
@shahmishal ping

**Comment by compnerd** [comment]
@swift-ci please test

**Comment by compnerd** [comment]
@shahmishal @tomerd - can we get this merged?

**Review by shahmishal — APPROVED** [review]

**Pull request merged**